educator/tests/Test/Dscp/Educator/Web/Student/Queries.hs

```diff
@@ -608,3 +612,10 @@ spec_Student_API_queries = specWithTempPostgresServer $ do
             let positiveGrades' = filter isPositiveGrade $ map _ptGrade txs
 
             return $ sort positiveGrades === sort positiveGrades'
+
+    describe "checkFairCVPDF" $ do
+        it "FairCV PDF validation works correctly" $ educatorProperty $ \info -> do
+            cert <- educatorAddCertificate info
+            _ <- updateMempoolWithPublications
+            check <- checkFairCVPDF cert
+            return (fairCVFullyValid $ fcacrCheckResult check)
```

Comment:
How about also check the negative case, i.e. flip a bit somewhere in PDF and make sure it does not validate anymore?